conda_smithy/configure_feedstock.py

```diff
@@ -1824,12 +1811,43 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
         },
         "github_actions": {
             "self_hosted": False,
+            "self_hosted_labels": [],
+            "self_hosted_triggers": ["push"],
+            "cancel_in_progress": False,
             # Set maximum parallel jobs
             "max_parallel": None,
             # Toggle creating artifacts for conda build_artifacts dir
             "store_build_artifacts": False,
             "artifact_retention_days": 14,
         },
+        "cirun_runners": [
+            {
+                "name": "cirun-openstack-gpu",
+                "labels": [
+                    "linux",
+                    "x64",
+                    "self-hosted",
+                    "cirun-openstack-gpu",
+                ],
+                "cloud": "openstack",
+                "instance_type": "gpu_large",
+                "machine_image": "ubuntu-2204-nvidia-container-runtime-20230107",
+                "region": "RegionOne",
+            },
+            {
+                "name": "cirun-openstack-cpu",
+                "labels": [
+                    "linux",
+                    "x64",
+                    "self-hosted",
+                    "cirun-openstack-cpu",
+                ],
+                "cloud": "openstack",
+                "instance_type": "ci_medium",
+                "machine_image": "ubuntu-2204-cloud-jammy-20221104",
+                "region": "RegionOne",
+            },
+        ],
```

Comment:
This should go in `conda-forge/.cirun`.

Reply:
Removed and created a PR here: https://github.com/conda-forge/.cirun/pull/1